lib/metalsmith.js

```diff
@@ -199,13 +198,16 @@ module.exports = metalsmith
   }))
 
   // render markdown in source files
-  .use(markdown({
+  .use(inPlace({
+    transform: 'jstransformer-marked',
     engineOptions: {
       breaks: true, // Enable line breaks
       mangle: false, // Don't mangle emails
       smartypants: true, // use "smart" typographic punctuation
       highlight: highlighter,
-      renderer: new DesignSystemRenderer() // Markdown renderer
+
+      // Custom markdown renderer
+      renderer: new DesignSystemRenderer()
```

Comment:
Our custom renderer uses a separate `marked` install - we had some issues in the past around keeping that in sync with `jstransformer-marked`. Right now, that's using v4.3.0 of `marked`, whereas the custom renderer is using v5.0.2.

Though I guess `metalsmith-markdown` also uses an internal version of `marked` (v4.3.0 at the mo), so it's probably not a problem. The explicit `transform` option here is really helpful in explaining what's using what here, so hopefully any issues would be easier to debug in the future.

Reply:
@domoscargin Good point. Shall I drop that last commit, one for another day?

Reply:
Brill, will do

For reference, the dependencies are still the same (which helps)

### Before
```console
├─┬ @metalsmith/markdown@1.10.0
│ └── marked@4.3.0
└── marked@5.0.2
```

### After
```console
├─┬ jstransformer-marked@1.4.0
│ └── marked@4.3.0
└── marked@5.0.2
```